slither.config.json

```diff
@@ -1,3 +1,4 @@
 {
-  "detectors_to_exclude": "similar-names,naming-convention,solc-version,timestamp,reentrancy-events,reentrancy-benign,reentrancy-no-eth"
+  "detectors_to_exclude": "similar-names,naming-convention,solc-version,reentrancy-events,timestamp,too-many-digits,pragma",
+  "filter_paths": "node_modules|contracts/test|contracts/mock|contracts/integrations"
```

Comment:
whoever reviews, make sure you agree on this list of ignored directories and muted checkers

Reply:
https://github.com/sandclock-org/solidity_contracts/pull/2/commits/4014cd4bac4f4731897e3fe24f8b2208b6ad4234